PORTAL/jobs/_pyperformance.py

```diff
@@ -1115,7 +1118,8 @@ def _parse_ignored(
             lines: Iterator[str],
             required: bool = True
     ) -> Tuple[Optional[List[str]], Optional[str]]:
-        # Ignored benchmarks (2) of benchmark-results/cpython-3.10.4-9d38120e33-fc_linux-42d6dd4409cb.json: genshi_text, genshi_xml
+        # Ignored benchmarks (2) of cpython-3.10.4-9d38120e33-fc_linux-42d6dd4409cb.json:
+        #   genshi_text, genshi_xml
```

Comment:
This is actually copy-and-pasted example output so it would be good to leave it as-is.